### PR TITLE
Code Cleanup

### PR DESCRIPTION
**Pull request by SkyHearn**
Current changes are:
- Fixing bugs and edge cases 
- removing unnecessary serial printouts 
- Easily switching between esp8266 and esp32

### CODE REVIEW

File: src/network.cpp
Comment (HumanInfluenced) [inline src/network.cpp:1]: C style cast on line 107

Comment (SkyHearn) [reply src/network.cpp:1]: Alright, I switched it to a reinterpret cast.

**Review by HumanInfluenced — COMMENTED** [review]
:-)

**Review by HumanInfluenced — APPROVED** [review]
:))

**Pull request merged**